### PR TITLE
✨RELEASE 1.0 changelog and readme cleanup.

### LINKED ISSUE #267: Stravalib RELEASE 1.0  --> PyPI
Hi everyone. 
This will be our first release in a really long time. 
And we have grown our maintainer team which is allowing us to chip away at more issues! 🎆 

[Click here to see  what stravalib will look like on PyPI](https://test.pypi.org/project/stravalib/). You can also install it to test version using:

`pip install -i https://test.pypi.org/simple/ stravalib`

I think the following TODO's are left
- [ ] Update the changelog to include fixes, updates and contributors to this release 
- [ ] Merge that PR
- [ ] Create a tagged release 1.0
- [ ] That release/tag will then trigger a deployment to pypi
- [ ] TEST the pypi deployment as it will be the first deployment in a long time and using a new build and package structure

-------
Note once we have done this I plan to submit a documentation PR that will clarify all of this in our docs. 

Am I missing anything? are we good to release stravalib v1.0 ?

### PR DESCRIPTION
**Pull request by lwasser**
closes #267 

@jsamoocha am i missing some issues / PR's here in the changelog?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,8 +97,15 @@ to-do list.
 
 ### Authentication
 
-In order to make use of this library, you will need to have access keys for one or more Strava users. This
-effectively requires you to run a webserver; this is outside the scope of this library, but stravalib does provide helper methods to make it easier.
+In order to make use of this library, you will need to create an app in Strava 
+which is free to do. Have a look at this tutorial for instructions on creating 
+an app with Strava. 
+
```
Comment (lwasser) [inline README.md:103]:
```suggestion
which is free to do. [Have a look at this tutorial for instructions on creating 
an app with Strava - we will be updating our docs with this information soon. ]([✨](https://medium.com/analytics-vidhya/accessing-user-data-via-the-strava-api-using-stravalib-d5bee7fdde17)

```

File: README.md [hunk 2]
```diff
@@ -96,8 +97,15 @@ to-do list.
 
 ### Authentication
 
-In order to make use of this library, you will need to have access keys for one or more Strava users. This
-effectively requires you to run a webserver; this is outside the scope of this library, but stravalib does provide helper methods to make it easier.
+In order to make use of this library, you will need to create an app in Strava 
+which is free to do. [Have a look at this tutorial for instructions on creating 
+an app with Strava - we will be updating our docs with this information soon. ]([✨](https://medium.com/analytics-vidhya/accessing-user-data-via-the-strava-api-using-stravalib-d5bee7fdde17)
```
Comment (lwasser) [inline README.md:102]:
```suggestion
which is free to do. [Have a look at this tutorial for instructions on creating 
an app with Strava - we will be updating our docs with this information soon.](https://medium.com/analytics-vidhya/accessing-user-data-via-the-strava-api-using-stravalib-d5bee7fdde17)
```

File: README.md [hunk 3]
```diff
@@ -1,30 +1,27 @@
 # Welcome to stravalib
 
-The **stravalib** project aims to provide a simple API for interacting with Strava v3 web services, in particular
-abstracting the v3 REST API around a rich and easy-to-use object model and providing support for date/time/temporal attributes
-and quantities with units (using the [python units library](http://pypi.python.org/pypi/units)).
+The **stravalib** Python package provides easy-to-user= tools for accessing and
```
Comment (jsamoocha) [inline README.md:3]: Maybe remove "easy-to-user= "?

**Review by jsamoocha — APPROVED** [review]
other than the typo in README, LGTM

File: README.md (hunk 3 above)
Comment (lwasser) [reply README.md:3]: oops thanks will fix now.

Comment (lwasser) [inline README.md:3]:
```suggestion
The **stravalib** Python package provides easy-to-use tools for accessing and 
```

**Comment by lwasser** [comment]
thanks @jsamoocha for the typo catch and review!! i'll merge and make a release. let's see if the build works. 🤞

**Pull request merged**